docs/api/speech_challenges.md

```diff
@@ -101,6 +101,7 @@ Name             | Type     | Description
 `referenceAudio` | `blob`   | **Optional** The reference audio fragment.
 `srt`            | `blob`   | **Optional** The transcription of the challenge in `.srt` format.
 `image`          | `blob`   | **Optional** An image to show with the challenge.
```

Comment:
I expected this field to be added to a speech recording, not challenge, as the ASR output is per recording, right?

Reply:
You must be mixing up the intention of this change. This is just a field that already exists but wasn't documented yet. #82 adds the field you expect.